src/components/form/EncounterFormPage.jsx

```diff
@@ -135,7 +135,7 @@ class EncounterFormPage extends React.PureComponent {
           (<div>
             <EncounterForm
               defaultValues={this.props.defaultValues}
-              encounter={this.getForm().encounter}
+              encounter={this.getForm().encounter || this.props.encounter}
```

Comment:
Not sure if you need this?

If you pass in an encounter uuid in the format:

```
{  
   encounter: {
       uuid: 'abc'
   }
}
```
... then this page should handle loading the encounter that backs the form for you...

It looks like here the encounter has already been fetched and you are passing the entire encounter in?

It's possible we want to support this as well, but note that the EncounterFormPage is still going to try to re-fetch the encounter if an encounter is passed in... see line 75 in EncounterFormPage.

Reply:
Going to go ahead and merge this in, though I'm still not 100% sure it's required.

I will confirm that it doesn't break the other apps that use react-components...